### PR TITLE
add optional notification duration

### PR DESCRIPTION
**Pull request by koush**

### CODE REVIEW

**Comment by koush** [comment]
my setInterval commit may be sketchy. I'm not sure the reasoning behind why it was implemented that way, but my commit is an alternative.

File: lib/assets/notification.css [hunk 1]
```diff
@@ -151,3 +151,10 @@ html, body {
   background: #147EFB;
   color: #fff;
 }
+.flat, .flat:before,
+#notification.flat, #notification.flat:before,
+#notification #buttons.flat, #notification #buttons.flat:before,
+#notification .icon.flat, #notification .icon.flat:before,
+#notification .message.flat, #notification .message.flat:before  {
+  border-radius: 0px;
```
Comment (bayleedev) [inline lib/assets/notification.css:159]: This feels redundant. Wouldn't line 154 be enough?

Also what do you think about just putting `flat` on the top level element and just targeting it with `.flat thing`?

File: lib/notifier.js [hunk 2]
```diff
@@ -8,9 +8,6 @@ class Notifier {
     } else {
       this.BrowserWindow = electron.BrowserWindow
     }
-    this.interval = setInterval(() => {
-      this.tick()
-    }, 100)
```
Comment (bayleedev) [inline lib/notifier.js:13]: Nice way around using an interval!

File: lib/assets/notification.css (hunk 1 above)
Comment (koush) [reply lib/assets/notification.css:159]: I tried just line 154, but the CSS was being clobbered by subsequent more specific selectors it seems. Give it a go. I'm no CSS expert. :)

**Comment by davidcoleman007** [comment]
Hi guys I need this functionality.  Can this be merged please?  Or possibly separate it into 2 pulls, since the .flat css is obviously not related to the working changes for duration?  Thanks in advance!  I can do the work if you guys don't have the time for it.

**Comment by bayleedev** [comment]
Deployed as `v0.1.2`

**Comment by davidcoleman007** [comment]
Thank you guys!!!

**Comment by davidcoleman007** [comment]
@blainesch I forked this repo last night and added some simple stacking logic too.  feel free to check it out and let me know if it's something u want in the project.  It (my stacking logic) could certainly be MUCH better, but it is what it is.

**Comment by bayleedev** [comment]
@davidcoleman007 I'm happy to merge in almost anything. PR's are appreciated :D

**Comment by koush** [comment]
@davidcoleman007 +1 on stacking.

**Comment by davidcoleman007** [comment]
Hey guys, I'll make a pr from my fork tonight.  It's nothing special but it's a step in the right direction.

**Pull request merged**